graylog2-web-interface/src/components/lookup-tables/LookupTablesOverview.jsx

```diff
@@ -22,41 +22,96 @@ import Routes from 'routing/Routes';
 import { Row, Col, Table, Popover, Button } from 'components/bootstrap';
 import { OverlayTrigger, PaginatedList, SearchForm, Icon } from 'components/common';
 import LUTTableEntry from 'components/lookup-tables/LUTTableEntry';
+import withPaginationQueryParameter from 'components/common/withPaginationQueryParameter';
 import { LookupTablesActions } from 'stores/lookup-tables/LookupTablesStore';
 
 import Styles from './Overview.css';
 
+const _lookupName = (id, map) => {
+  const empty = { title: 'None' };
+
+  if (!map) {
+    return empty;
+  }
+
+  return map[id] || empty;
+};
+
+const _helpPopover = () => {
```

Comment:
**Informational:** This pattern is repeated a couple of times in older code like this. The `_helpPopover` function is actually a React component in disguise. Therefore we should just treat it as one and name it `HelpPopover` and render it as a component (`<HelpPopover />`) instead of calling the function.

I am not saying you should change this in this PR, extracting a stateless function like this to the module scope is great already. But maybe if you stumble across patterns like this in the future, you can turn them into components straight-away.